packages/examples/packages/send-flow/src/components/SendForm.tsx

```diff
@@ -69,21 +65,12 @@ export const SendForm: SnapComponent<SendFormProps> = ({
       </Box>
     </Field>
     <Field label="To account" error={errors?.to}>
-      <Box>
-        <Image src={jazzicon3} />
-      </Box>
-      <Input
+      <AddressInput
         name="to"
+        chainId="eip155:0"
         placeholder="Enter receiving address"
         value={flushToAddress ? '' : undefined}
       />
-      {displayClearIcon && (
```

Comment:
Why did you remove this?

Reply:
The `AddressInput` component has a clear icon built-in.